src/6_branch.js

```diff
@@ -1013,7 +1013,7 @@ Branch.prototype['sendSMS'] = wrap(
 					self._api(
 						resources.linkClick,
 						{
-							"link_url": 'l/' + url.split('/').pop(),
```

Comment:
ah, happy to explain!

as you have seen above, the "before" logic was, oh what a link gets returned from the api backend? TREAT IT AS AN L LINK!

This is very inconvenient to what you are trying to do in api backend, because now, when the v1/url returns `Z2Qa/a1b2c3d4`, the innocent old Web SDK will try to hit `bnc.lt/l//a1b2c3d4` which will make the link service return an equivalent of 404.

So what I did here is to hit `l/...` for l links, `Z2Qa/...` for Z2Qa/ links, `AB12/...` for AB12 links. Or if you will, to make it more general.